src/java.desktop/share/native/libharfbuzz/hb-object.hh

```diff
@@ -325,7 +325,7 @@ retry:
   hb_user_data_array_t *user_data = obj->header.user_data.get_acquire ();
   if (unlikely (!user_data))
   {
-    user_data = (hb_user_data_array_t *) hb_calloc (sizeof (hb_user_data_array_t), 1);
```

Comment:
Note 
(1) We don't cherry pick unless it is critical.
(2) I doubt you have ownership of harfbuzz code such that you can contribute any part of it to JDK in accordance with the OCA you signed.

Accordingly, the right thing to do here is disable the warning for this library and tool chain.

Reply:
Sure, I wasn't exactly hiding that it was cherry-picked. I did not consider (2) to be an issue given OpenJDK wholesale imports releases, but w/e.

Anyway, happy to do that. Can you suggest what the idiomatic way of doing that in the OpenJDK codebase is for imported libs?

Reply:
add the warning here[
](https://github.com/openjdk/jdk/blob/8b9bf758801400e4491326cd4c90fc117b9d97e1/make/modules/java.desktop/lib/Awt2dLibraries.gmk#L506)

Reply:
PS make sure you include a comment which toolchain / version required this